tests/utils/guidance_test.py

```diff
@@ -0,0 +1,143 @@
+from client.protocols import unlock
+from client.utils import guidance
+from client.sources.common import models
+from client.utils import assess_id_util
+import mock
+import unittest
+import os
+
+
+class GuidanceProtocolTest(unittest.TestCase):
+    GUIDANCE_DIRECTORY = "demo/guidance_test/"
+    MISUCOUNT_FILE = "tests/misUcount.json"
+    TEST = "Test"
+    CASE_ID = TEST + ' > Suite 1 > Case 1 > Prompt 1'
+    UNIQUE_ID = "\nTest\n\n>>> 1 + 1 # OK will accept 'black' as right answer\ne74918d4310bb6cbc896676f20dc20de\n# locked\n>>> 2 + 2 # OK will accept 'black' as right answer\ne74918d4310bb6cbc896676f20dc20de\n# locked\n"
```

Comment:
Technically though, the test should be giving it proper input, unless that is part of the test. Please run this through the canonicalizer and put the canonicalized version here.